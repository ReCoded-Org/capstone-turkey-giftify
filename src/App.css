.mainContainer {
  padding: 0 !important;
  background-color: #0077b6;
}