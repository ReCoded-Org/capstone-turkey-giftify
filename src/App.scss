@import "~bootstrap/scss/bootstrap";

.container-fluid {
    padding: 0 !important;
    background-color: #0077B6;

}