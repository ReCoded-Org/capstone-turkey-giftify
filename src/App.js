import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import { Container } from "react-bootstrap";
import { Route } from "react-router";
import { BrowserRouter as Router } from "react-router-dom";
import "./App.css";
import Footer from "./components/Footer/Footer";
import Navbar from "./components/Navbar/Navbar";
import HomePage from "./containers/HomePage/HomePage";
import RequestItem from "./containers/RequestItem/RequestItem";

function App() {
  return (
    <>
      <Container fluid className="mainContainer">
        <Router>
          <Route path="/" component={Navbar} />
          <Route exact path="/" component={HomePage} />
          <Route exact path="/request-item/:id" component={RequestItem} />
          {/* <Route path="/requests" component={Requests} />
          <Route path="/donations" component={Donations} />
          <Route path="/about" component={About} />
          <Route path="/contactUs" component={ContactUs} /> */}
          <Route path="/" component={Footer} />
        </Router>
      </Container>
    </>
  );
}

export default App;
