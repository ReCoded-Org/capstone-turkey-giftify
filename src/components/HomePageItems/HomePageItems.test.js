import React from "react";
import renderer from "react-test-renderer";
import App from "../../App";

it("HomePageItems renders correctly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
