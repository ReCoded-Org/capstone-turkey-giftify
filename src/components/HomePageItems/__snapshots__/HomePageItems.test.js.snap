// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mainContainer container-fluid"
>
  <div
    className="container"
  >
    <div
      className="homePageItems row row-cols-xl-4 row-cols-md-3 row-cols-2"
    />
  </div>
</div>
`;
